hipify_torch/cuda_to_hip_mappings.py

```diff
@@ -8938,7 +8938,7 @@
         # This substitution is not permissible, because there's another copy of this
         # function in torch/cuda.h
         # ("cuda::device_count", ("hip::device_count", API_C10)),
-        ("cuda::current_device", ("hip::current_device", API_C10)),
+        ("at::cuda::current_device", ("at::hip::current_device", API_C10)),
```

Comment:
doesnt satisfy - https://github.com/ROCmSoftwarePlatform/pytorch/blob/rocm6.0_internal_testing/aten/src/ATen/cuda/CUDAContext.cpp#L43

Reply:
As well as previous commented rule does not satisfy https://github.com/ROCmSoftwarePlatform/pytorch/blob/rocm6.0_internal_testing/aten/src/ATen/cuda/CUDAContext.cpp#L20
All c10::cuda cases are covered at line 8935